recipes/public-wrappers/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "public-wrappers" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "f2173dc480ea8b2baf59b2811d19783ea86005aabf2a7fdbb284a2a752d04a6a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/tesujimath/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [win]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Can you switch to the new standard of using `pip`. You'll need to add `pip` as a `build` requirment:

https://conda-forge.org/docs/meta.html#use-pip

Reply:
OK, missed that before sorry.